### ISSUE TITLE
`alterables` of `Seeker` class can be changed to allow illegal modification of Seeker data

### ISSUE THREAD

**Issue by BelTol3011** [seekers-dev/seekers#1]
The `alterables` attribute of the `Seeker` class can be modified (since `copy.deepcopy` doesn't copy the class) and allows changing any attribute of the seekers.

```py
Seeker().__class__.alterables = (
    ('target', Vector.is_vector),
    ('magnet', Magnet.is_magnet),
    ('position', lambda: True)
)
```

The radius and other class attributes can also be changed:
```py
Seeker().__class__.radius = 50
```

**Comment by BelTol3011** [comment]
This is fixed on the [Belissimo](https://github.com/seekers-dev/seekers/tree/Belissimo)-branch using gRPC

**Comment by BelTol3011** [comment]
Closed in #6
